site/content/docs/5.2/components/toasts.md

```diff
@@ -249,7 +249,7 @@ For systems that generate more notifications, consider using a wrapping element
         {{< placeholder width="20" height="20" background="#ff7900" class="me-2" text="false" title="false" >}}
         <strong class="me-auto">Boosted</strong>
         <small>just now</small>
-        <button type="button" class="btn-close ms-2" data-bs-dismiss="toast"><span class="visually-hidden">Close</span></button>
+        <button type="button" class="btn-close ms-2" data-bs-dismiss="toast" data-bs-toggle="tooltip" data-bs-placement="bottom" data-bs-title="Close"><span class="visually-hidden">Close</span></button>
```

Comment:
z-index problem when using a wrapping container (see [http://https://deploy-preview-1823--boosted.netlify.app/docs/5.2/components/toasts/#placement](http://https//deploy-preview-1823--boosted.netlify.app/docs/5.2/components/toasts/#placement)). Toast container z-index is 1090 and tooltip z-index is 1080